### PR TITLE
Replace the SBT lesson with Maven

### PR DESCRIPTION
**Pull request by matteosb**

### CODE REVIEW

**Comment by matteosb** [comment]
@stephbian

File: web/maven.textile [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+
+We'll be creating a simple JSON parser for simple tweets.  Add the following code to
+@src/main/scala/sample/SimpleParser.scala@
+
+<pre>
+package sample
+
+case class SimpleParsed(id: Long, text: String)
+
+class SimpleParser {
```
Comment (stephbian) [inline web/maven.textile:98]: object?

File: web/maven.textile [hunk 2]
```diff
@@ -0,0 +1,458 @@
...
+scala> parser.parse(tweet)
+res0: Option[sample.SimpleParsed] = Some(SimpleParsed(1,foo))
+
+scala>
+</pre>
+
+h2. Adding Tests
+
+The Maven build you pulled down already has our testing library,
+ScalaTest, declared as a dependency. So Let's add a spec in
```
Comment (stephbian) [inline web/maven.textile]: lowercase "Let's"

File: web/maven.textile [hunk 3]
```diff
@@ -0,0 +1,458 @@
...
+class SimpleParserSpec extends WordSpec with Matchers {
+
+  "SimpleParser" should {
+    val parser = new SimpleParser()
+
+    "work with basic tweet" in {
+      val tweet = """{"id":1,"text":"foo"}"""
+      parser.parse(tweet) match {
+        case Some(parsed) =>
+	  parsed.text should equal("foo")
```
Comment (stephbian) [inline web/maven.textile]: fix spacing

**Pull request merged**